packages/cli/lib/sandboxes.js

```diff
@@ -14,53 +14,49 @@ const { getHubSpotWebsiteOrigin } = require('@hubspot/local-dev-lib/urls');
 const {
   HUBSPOT_ACCOUNT_TYPES,
 } = require('@hubspot/local-dev-lib/constants/config');
-const {
-  STANDARD_SANDBOX_TYPE,
-  DEVELOPER_SANDBOX_TYPE,
-  DEVELOPER_SANDBOX,
-  STANDARD_SANDBOX,
-} = require('./constants');
 
 const syncTypes = {
   OBJECT_RECORDS: 'object-records',
 };
 
 const sandboxTypeMap = {
-  DEV: DEVELOPER_SANDBOX_TYPE,
-  dev: DEVELOPER_SANDBOX_TYPE,
-  DEVELOPER: DEVELOPER_SANDBOX_TYPE,
-  developer: DEVELOPER_SANDBOX_TYPE,
-  DEVELOPMENT: DEVELOPER_SANDBOX_TYPE,
-  development: DEVELOPER_SANDBOX_TYPE,
-  STANDARD: STANDARD_SANDBOX_TYPE,
-  standard: STANDARD_SANDBOX_TYPE,
+  DEV: HUBSPOT_ACCOUNT_TYPES.DEVELOPMENT_SANDBOX,
+  dev: HUBSPOT_ACCOUNT_TYPES.DEVELOPMENT_SANDBOX,
+  DEVELOPER: HUBSPOT_ACCOUNT_TYPES.DEVELOPMENT_SANDBOX,
+  developer: HUBSPOT_ACCOUNT_TYPES.DEVELOPMENT_SANDBOX,
+  DEVELOPMENT: HUBSPOT_ACCOUNT_TYPES.DEVELOPMENT_SANDBOX,
+  development: HUBSPOT_ACCOUNT_TYPES.DEVELOPMENT_SANDBOX,
+  STANDARD: HUBSPOT_ACCOUNT_TYPES.STANDARD_SANDBOX,
+  standard: HUBSPOT_ACCOUNT_TYPES.STANDARD_SANDBOX,
```

Comment:
Keeping this map here for users who use the --type flag where they can type whatever they want. This just handles random strings that they may type, but let me know if this is overkill and we can just remove it

Reply:
Hmm,  think having the different variations of "developer" makes sense, but we could also `toLowerCase` or `toUpperCase` any strings that we're using as keys for this and not need the different capitalizations

Reply:
good call, pushed an update with this and the lang key change!